testing/src/main/java/org/oppia/android/testing/KonfettiViewMatcher.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.android.testing
+
+import android.view.View
+import nl.dionsegijn.konfetti.KonfettiView
+import org.hamcrest.Description
+import org.hamcrest.TypeSafeMatcher
+
+class KonfettiViewMatcher {
+  companion object {
+    // Returns a matcher that matches for active confetti.
```

Comment:
Please convert to KDoc & please make sure it's framed as documentation (e.g. it should explain what the thing is to help other developers when they are deciding whether to use this matcher). Please also add documentation for the class.

Reply:
Done